pysilcam/tests/test_z_classify.py

```diff
@@ -65,6 +67,11 @@ def correct_positives(category):
 
         # turn failed count into a success percent
         success = 100 - (failed / len(files)) * 100
+
+        t2 = pd.Timestamp.now()
+        td = t2 - t1
+        assert td < pd.to_timedelta(time_limit), 'Processing time too long.'
```

Comment:
to_timedelta uses nanoseconds as default time unit, probably not was intended here (pass unit='seconds' or similar to be explicit).